splinterd/src/config/mod.rs

```diff
@@ -459,12 +459,20 @@ mod tests {
     static EXAMPLE_NODE_ID: &str = "012";
     static EXAMPLE_DISPLAY_NAME: &str = "Node 1";
 
+    /// Default configuration values
     static DEFAULT_CLIENT_CERT: &str = "client.crt";
     static DEFAULT_CLIENT_KEY: &str = "client.key";
     static DEFAULT_SERVER_CERT: &str = "server.crt";
     static DEFAULT_SERVER_KEY: &str = "server.key";
     static DEFAULT_CA_CERT: &str = "ca.pem";
 
+    static DEFAULT_STATE_DIR: &str = "/var/lib/splinter/";
```

Comment:
The changes here seem to be unrelated to the commit

Reply:
I added a bit to the commit message to explain why the defaults were added. Mainly, because the node_id is now in the DefaultPartialConfigBuilder, a Config object can be successfully built from just that. Therefore, the test which previously errored because of a missing node_id, now passes and is able to create a Config object. So, adding all of these defaults is a way to verify the contents of that Config object.